src/Datadog.Trace.ClrProfiler.Native/cor_profiler.cpp

```diff
@@ -301,57 +271,68 @@ HRESULT STDMETHODCALLTYPE CorProfiler::AssemblyLoadFinished(AssemblyID assembly_
 
   const auto assembly_info = GetAssemblyInfo(this->info_, assembly_id);
   if (!assembly_info.IsValid()) {
+    Debug("AssemblyLoadFinished: ", assembly_id, " ", hr_status);
     return S_OK;
   }
 
-  ComPtr<IUnknown> metadata_interfaces;
-  auto hr = this->info_->GetModuleMetaData(assembly_info.manifest_module_id, ofRead | ofWrite,
-                                           IID_IMetaDataImport2,
-                                           metadata_interfaces.GetAddressOf());
+  const auto is_managed_profiler = assembly_info.name == WStr("Datadog.Trace.ClrProfiler.Managed");
```

Comment:
We should change this name to avoid conflicts with Continuous Profiler. Maybe something like `is_instrumentation_assembly`?

Reply:
Even without Continuous Profiler, "managed profiler" was not correct even though we use it in several places 😞  The _clr profiler_ has two components: a native library and a managed library.

Reply:
sounds good to me, will do.

Reply:
done in 461a6c5